src/Directory.Build.props

```diff
@@ -1,54 +0,0 @@
-<Project>
-  <!-- Enable multi-level merging -->
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))"/>
-
-  <PropertyGroup>
-    <Version>10.0.0</Version>
-    <AssemblyVersion>10.0.0</AssemblyVersion>
-    <InformationalVersion>10.0.0-rc1</InformationalVersion>
-    <FileVersion>10.0.0</FileVersion>
-    <LangVersion Condition="'$(LangVersion)' == ''">preview</LangVersion>
-    <NeutralLanguage>en-US</NeutralLanguage>
-    <Company>Umbraco CMS</Company>
-    <Copyright>Copyright © Umbraco 2021</Copyright>
-    <Authors>Umbraco HQ</Authors>
-    <PackageProjectUrl>https://umbraco.com/</PackageProjectUrl>
-    <PackageIconUrl>https://umbraco.com/dist/nuget/logo-small.png</PackageIconUrl>
-    <PackageLicenseUrl>https://opensource.org/licenses/MIT</PackageLicenseUrl>
-    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <PackageTags>umbraco</PackageTags>
-    <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://github.com/umbraco/umbraco-cms</RepositoryUrl>
-    <Nullable>enable</Nullable>
-    <WarningsAsErrors>Nullable</WarningsAsErrors>
-    <ImplicitUsings>enable</ImplicitUsings>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <!-- SourceLink: Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element)-->
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
-
-    <!-- SourceLink: Embed source files that are not tracked by the source control manager in the PDB -->
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
-
-    <!-- SourceLink: Build symbol package (.snupkg) to distribute the PDB containing Source Link -->
-    <IncludeSymbols>true</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-  </PropertyGroup>
-
-  <!-- SourceLink: Deterministic -->
-  <!-- https://github.com/clairernovotny/DeterministicBuilds -->
-  <!-- Only for Azure Pipelines CI Build -->
-  <PropertyGroup Condition="'$(TF_BUILD)' == 'true'">
```

Comment:
The `TF_BUILD` system variable is specific to Azure Pipelines (previously Team Foundation Server), but we shouldn't add conditions specific to certain build environments into our shared project properties. The recommended way is to include `-p:ContinuousIntegrationBuild=true` in the actual CI/CD build script, hence the changed to `build\azure-pipelines.yml`.